src/index.js

```diff
@@ -347,7 +347,7 @@ function createConfig(options, entry, format, writeMeta) {
 	// We want to silence rollup warnings for node builtins as we rollup-node-resolve threats them as externals anyway
 	// @see https://github.com/rollup/plugins/tree/master/packages/node-resolve/#resolving-built-ins-like-fs
 	if (options.target === 'node') {
-		external = external.concat(builtinModules);
+		external = [/node:.*/].concat(builtinModules);
```

Comment:
I purposefully replaced `external` here as otherwise we get duplicates.

https://github.com/developit/microbundle/blob/6c59174aadefb5ac5cf5e7670a750c4175b1c0d5/src/index.js#L339

Unless this was intentional as slightly faster path for those 4 builtins specifically?